src/sentry/issues/grouptype.py

```diff
@@ -398,6 +398,18 @@ class PerformanceP95EndpointRegressionGroupType(GroupType):
     notification_config = NotificationConfig(context=["Users Affected", "State", "Regressed Date"])
 
 
+# experimental
+@dataclass(frozen=True)
+class PerformanceStreamedSpansGroupTypeExperimental(GroupType):
```

Comment:
I see in the develop docs that options should be created automatically for this? All I have to do is add a flagr flag for the UI?

Reply:
Yep! Since we need to use options automator to update options, you'll unfortunately need to figure out the option keys yourself. Code that generates them is here: https://github.com/getsentry/getsentry/blob/416b44f97016489972bcfe93170bed52e0903a7b/getsentry/features.py#L1418-L1421

Alternatively, you can find them in https://sentry.io/_admin/options/ under your `<Category>:<Description>`. It's generally not recommended to use the admin ui to update these now, but you can at least get the option names for use in automator.